docs/deploying.md

```diff
@@ -369,6 +369,150 @@ support.
 		* Closing connection 0
 		curl: (60) SSL certificate problem: self signed certificate
 		More details here: http://curl.haxx.se/docs/sslcerts.html
+
+## Deploy a v2 registry with Nginx and basic authentication
+
+This example shows you how to securely deploy a v2 registry using basic authentication.
+
+### Install Docker Compose
+
+1. Open a new terminal on the host with your `distribution` directory.
+
+2. Get the `docker-compose` binary.
+
+		$ sudo wget https://github.com/docker/compose/releases/download/1.2.0/docker-compose-`uname  -s`-`uname -m` -O /usr/local/bin/docker-compose
+
+	This command installs the binary in the `/usr/local/bin` directory. 
+	
+3. Add executable permissions to the binary.
+
+		$  sudo chmod +x /usr/local/bin/docker-compose
+		
+
+### Do some housekeeping
+
+1. Remove any previous images.
+
+		$ docker rmi -f $(docker images -q -a )
+		
+	 This step is a house keeping step. It prevents you from mistakenly picking up
+	 an old image as you work through this example.
+		
+2. Edit the `distribution/cmd/registry/config.yml` file and remove the `tls` block.
+
+	If you worked through the previous example, you'll have a `tls` block. To use
+	TLS with the registry and Nginx, you should terminate the TLS on the Nginx
+	rather than the registry.
+
+### Configure your system
+
+1. Change to the `distribution/contrib/basicauth` directory.
+
+	This directory contains a single `docker-compose.yml` configuration file and an `nginix`
```

Comment:
Same here on `nginix`; if this is just a "custom" name, I'd pick something *really* different, to prevent confusion and people using the wrong image by mistake